docs/products/opensearch/reference/plugins.rst

```diff
@@ -20,13 +20,17 @@ These plugins are enabled on all Aiven for OpenSearch services by default:
 * `Index Management <https://opensearch.org/docs/im-plugin/index/>`__
 * `k-NN <https://opensearch.org/docs/search-plugins/knn/index/>`__
 * `Notebooks <https://opensearch.org/docs/dashboards/notebooks/>`__
```

Comment:
Nobtebooks is part of observability plugin. In which version it was moved I duno.